frontend/src/views/Login.js

```diff
@@ -45,6 +50,16 @@ const useStyles = makeStyles((theme) => ({
 function Login() {
 
   const classes = useStyles();
+  const dispatch = useDispatch();
+
+  const user = useSelector(userDetailsSelector);
+
+  useEffect(() => {
+    dispatch(getUserRequest(1));
+  }, [])
+
+  // Just Testing To Ensure it works
+  console.log(user);
```

Comment:
jsut testing to show that it works haha, will be removed when I make the dashboard